public/js/card.js

```diff
@@ -16,10 +15,6 @@ if (url.indexOf("?deck_id=") !== -1) {
   getPosts();
 }
 
-var getPosts = function() {},
-  displayEmpty = function() {},
-  initializeRows = function() {};
-
```

Comment:
preserve these lines to prevent travis failing, we can edit it after changes are merged